topo_processor/util/epsg.py

```diff
@@ -0,0 +1,6 @@
+from rasterio.crs import CRS
+
+
+def epsg_code(crs: CRS):
```

Comment:
there is no point for this function if you are just using rasterio.

Reply:
true

Reply:
I will remove it